plugins/woocommerce-admin/client/products/fields/attribute-field/attribute-field.tsx

```diff
@@ -33,6 +33,8 @@ type AttributeFieldProps = {
 	value: ProductAttribute[];
 	onChange: ( value: ProductAttribute[] ) => void;
 	productId?: number;
+	// TODO: should we support an 'any' option to show all attributes?
+	attributeType?: 'regular' | 'for-variations';
```

Comment:
From a completeness standpoint, it feels like it would be useful to have a mode where we could see all attributes, regardless of how they were assigned to the product. But, we don't currently have any need for this in the UI, and it would add some additional work to properly support it, so *probably* best to just push this off until we (or a 3PD) actually needs it.